ansible/roles/k8s_master/tasks/init_master_leader.yml

```diff
@@ -1,26 +1,26 @@
 - name: Get IP Address for HAProxy Machine
   set_fact: 
     haproxy_ip="{{ hostvars[item].ansible_ens2.ipv4.address }}"
+    haproxy_node="{{ hostvars[item].inventory_hostname }}"
   when: hostvars[item].haproxy is defined
   with_items: 
   - "{{ groups['k8s_vms' + msetnumber + '_' + servernumber] }}"
 
 - name: Wait for HAProxy service to be ready
   wait_for:
-    port: 80
+    port: "{{ k8s_port }}"
     host: "{{ haproxy_ip }}"
     connect_timeout: 3
     delay: 3
     timeout: 300
 
 - name: Initializing Kubernetes cluster
-  shell: kubeadm init --control-plane-endpoint {{ haproxy_ip }}:80 --upload-certs --apiserver-advertise-address {{ ansible_ens2.ipv4.address }} --skip-phases=addon/kube-proxy --skip-phases=addon/coredns --ignore-preflight-errors=NumCPU
+  shell: kubeadm init --control-plane-endpoint {{ haproxy_ip }}:{{ k8s_port }} --upload-certs --apiserver-advertise-address {{ ansible_ens2.ipv4.address }} --skip-phases=addon/kube-proxy --skip-phases=addon/coredns
```

Comment:
>beadm init --control-plane-endpoint {{ haproxy_ip }}:{{ k8 [](start = 11, length = 58)

This line is too long to read. Could you split?

Reply:
Yes, I split it